.github/workflows/deploy.yml

```diff
@@ -46,7 +46,7 @@ jobs:
     - name: install
       uses: actions/setup-go@v3
       with:
-        go-version: 1.19
+        go-version: 1.20
 
```

Comment:
This needs to be wrapped in quotes to prevent it from being interpreted as a number and shortened to 1.2

Reply:
It might make sense to switch the actions to use the mod file and update the mod file to go 1.20, which I don't see done.

```
      uses: actions/setup-go@v4
      with:
        go-version-file: go.mod
```

Reply:
The go mod file has the minimum supported version. We might not want to use the minimum supported version in every case, we might prefer to use the newest available golang version instead.

Reply:
The go version in SDK in bumped in accordance to the supported K8s versions. We would intentionally not want to so beyond what k8s does. So making it to be compatible with `go.mod` would be a good idea.